.circleci/config.yml

```diff
@@ -611,6 +621,10 @@ commands:
           working_directory: go/pulumi
           name: yarn install go pulumi
           command: yarn install --immutable
+      - run:
```

Comment:
build volume reaper package (workspace is persisted across circle jobs, so we will have the built artifacts at docker build/push time)